app/templates/buyers/brief_publish_confirmation.html

```diff
@@ -34,20 +34,39 @@
     %}
         {% include "toolkit/page-heading.html" %}
     {% endwith %}
+
     {% if not published %}
-      <p class="padding-bottom-small">All requirements are published on the Digital Marketplace where anyone can see them.</p>
-      <p class="padding-bottom-small">All requirements are open for {{ dates.application_open_weeks }} weeks.</p>
-      <p class="padding-bottom-small">If you publish your requirements today ({{ dates.today | shortdateformat }}), suppliers will be able to apply until {{ dates.closing_time }}, {{ dates.closing_date | shortdateformat}}.</p>
+      <div class="dmspeak">
+        <p>All requirements are published on the Digital Marketplace where anyone can see them.</p>
+      </div>
+        {% if brief.lotSlug == 'digital-specialists' and not brief.requirementsLength %}
+          <p><a href="{{ url_for('.edit_brief_question', framework_slug=brief.frameworkSlug, lot_slug=brief.lotSlug, brief_id=brief.id, section_slug='set-how-long-your-requirements-will-be-live-for', question_id='requirementsLength') }}">Set how long your requirements will be open for</a></p>
+          <div class="dmspeak">
+            <p>This will show you what the supplier application deadline will be if you publish your requirements today.</p>
+          </div>
+        {% else %}
+          <p>Your requirements will be open for {{ dates.application_open_weeks }} {{ pluralize(dates.application_open_weeks, "week", "weeks") }}.</p>
```

Comment:
Made a comment about this [in the dm-utils PR](https://github.com/alphagov/digitalmarketplace-utils/pull/275/files#r72600783). Basically, it seems like we're recreating something we already have.